src/core/lib/event_engine/posix_engine/posix_engine.h

```diff
@@ -138,6 +139,9 @@ class PosixEventEngine final : public PosixEventEngineWithFdSupport,
  public:
   class PosixDNSResolver : public EventEngine::DNSResolver {
    public:
+    explicit PosixDNSResolver(ResolverOptions const& options,
```

Comment:
nit: In this codebase, we tend to write this as `const ResolverOptions& options`.